upgrade.py

```diff
@@ -448,6 +448,9 @@ def buildRestoreList(self):
         # NRPE service config
         self.restore_list += ['etc/nagios/nrpe.cfg', {'dir': 'etc/nrpe.d'}]
 
+        # Keep user multipath configuration
+        self.restore_list += [{'dir': 'etc/multipath/conf.d'}]
```

Comment:
Here we need to choose between either saving the whole directory, or just `custom.conf`. Saving the whole directory may be a perk when/if vendors start taking the habit of dropping a file of theirs into this directory when needed.

On the other hand, if we want to keep the liberty of packaging configuration files in `/etc/multipath/conf.d`, then it may be safer to just add `etc/multipath/conf.d/custom.conf` to the list.